app/helpers/reports_helper.rb

```diff
@@ -10,4 +10,19 @@ def report_options
       "Other"
     ]
   end
+
+  def report_model_property(report, model, property)
+    if report_has_model_property?(report, model, property)
+      return report.properties[model][property]
+    else
+      return ""
+    end
+  end
+
+  private
+  def report_has_model_property?(report, model, property)
+    return false unless report.properties[model]
+    return false unless report.properties[model].respond_to?(:[])
```

Comment:
Technically this line isn't necessary since old reports will fail the first check, but I am keeping it in just in case future additions store non-accessible content to `properties[model]`.